core/imageroot/var/lib/nethserver/cluster/actions/list-user-domains/50read

```diff
@@ -29,10 +29,7 @@ unconfigured_domains = []
 
 rdb = agent.redis_connect()
 
-internal_domains = cluster.userdomains.get_internal_domains(rdb) 
-external_domains = cluster.userdomains.get_external_domains(rdb)
-
-domains = external_domains | internal_domains
```

Comment:
So here the logic was correct already?

Reply:
Apparently yes.

We can debate what is the "correct" one, provided it is consistent.